sunpy/time/tests/test_time.py

```diff
@@ -27,13 +27,20 @@ def test_parse_time_24_2():
     assert dt.scale == 'utc'
 
 
-def test_parse_time_trailing_zeros():
-    # see issue #289 at https://github.com/sunpy/sunpy/issues/289
-    dt = parse_time('2010-10-10T00:00:00.00000000')
-    assert dt == Time('2010-10-10')
```

Comment:
Does this still work?

Reply:
Yes, but I replaced it with a "harder" test